src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/App/index.js

```diff
@@ -9,8 +9,13 @@ import LegacyContent from '../LegacyContent';
 import UnauthContent from '../UnauthContent';
 import featureFlagNames from 'platform/utilities/feature-toggles/featureFlagNames';
 import { selectPatientFacilities } from 'platform/user/selectors';
+import { isAuthenticatedWithSSOe } from 'platform/user/authentication/selectors';
 
-export const App = ({ facilities, showNewSecureMessagingPage }) => {
+export const App = ({
+  facilities,
+  showNewSecureMessagingPage,
+  authenticatedWithSSOe,
```

Comment:
Could you add this to PropTypes? https://github.com/department-of-veterans-affairs/vets-website/pull/15497/files#diff-1d106f7e3518dc1d13a6de0008fa0cb1a4a4cb965da671254fd10c57dd38dd18R50